src/messages.ts

```diff
@@ -7,6 +7,7 @@ export const Messages = {
     ConstructorSpecialMethod: 'Class constructor may not be an accessor',
     DeclarationMissingInitializer: 'Missing initializer in %0 declaration',
     DefaultRestParameter: 'Unexpected token =',
+    DefaultRestProperty: 'Unexpected token =',
```

Comment:
There is still no V8 error message for this, so we stick with a generic for the mean time.